packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -480,14 +501,11 @@ export class EngineReportingAgent<TContext = any> {
     }
 
     if (!documentAST) {
```

Comment:
I think we should be able to drop this conditional due to the non-`?` type declaration, though see my other comment.